### PR TITLE
add 2D convexhull and skeleton features to conservation tracking and …

### PR DESCRIPTION
**Pull request by ilastikdev**
…animal tracking

This has also been added in Structured Learning Workflow (branch: learning)

Issue: #1188

### CODE REVIEW

File: ilastik/workflows/tracking/conservation/animalConservationTrackingWorkflow.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def connectLane(self, laneIndex):
            
         super(AnimalConservationTrackingWorkflowBase, self).connectLane(laneIndex) 
         
-    def _createDivisionDetectionApplet(self):
+    def _createDivisionDetectionApplet(self,selectedFeatures=None):
```
Comment (akreshuk) [inline ilastik/workflows/tracking/conservation/animalConservationTrackingWorkflow.py]: shouldn't that be selectedFeatures=dict()?

Comment (ilastikdev) [inline ilastik/workflows/tracking/conservation/animalConservationTrackingWorkflow.py]: Code calls the proc in conservationTracking, where it is set to dict() so in the current code either works.

**Comment by akreshuk** [comment]
I tried object classification with this branch. When I choose features from "Standard" plugin (all for a 2d image) in object extraction and then open the object classification applet and click "Subset features", other plugins (like convex hull features, etc) are still visible, although they don't have any features listed under them. Would it be possible to remove them?

**Comment by ilastikdev** [comment]
This was already discussed  a year ago with Uli and it leads to users forgetting that there are other features that they could use once they limit only to one of the sets.

Also, when, for example, selecting a subset of Standard features we also display all of them with selected checked, so here too, we display all possible and mark the selected ones.

**Pull request merged**